eclair-core/src/main/scala/fr/acinq/eclair/db/PaymentsDb.scala

```diff
@@ -39,6 +39,7 @@ trait PaymentsDb {
 
   def getPaymentRequest(paymentHash: ByteVector32): Option[PaymentRequest]
 
+  // returns non paid payment request
```

Comment:
I don't think that comment is true, it will return all payment requests included the ones that have been paid

Reply:
In [SqlitePaymentDb#L158](https://github.com/ACINQ/eclair/blob/e128992c50022243b0609662ef7c39ae80cc2c8b/eclair-core/src/main/scala/fr/acinq/eclair/db/sqlite/SqlitePaymentsDb.scala#L158) we filter for `received_at IS NULL` so this retrieves only non paid payment requests, note that we update the record setting `received_at` when receiving a payment.

Reply:
Ah ok my bad.... I think we're good to go!